runners/core-construction-java/src/main/java/org/apache/beam/runners/core/construction/Timer.java

```diff
@@ -0,0 +1,131 @@
+/*
+ * Licensed to the Apache Software Foundation (ASF) under one
+ * or more contributor license agreements.  See the NOTICE file
+ * distributed with this work for additional information
+ * regarding copyright ownership.  The ASF licenses this file
+ * to you under the Apache License, Version 2.0 (the
+ * "License"); you may not use this file except in compliance
+ * with the License.  You may obtain a copy of the License at
+ *
+ *     http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+package org.apache.beam.runners.core.construction;
+
+import com.google.auto.value.AutoValue;
+import java.io.IOException;
+import java.io.InputStream;
+import java.io.OutputStream;
+import java.util.Collections;
+import java.util.List;
+import javax.annotation.Nullable;
+import org.apache.beam.sdk.coders.CoderException;
+import org.apache.beam.sdk.coders.InstantCoder;
+import org.apache.beam.sdk.coders.StructuredCoder;
+import org.apache.beam.sdk.util.common.ElementByteSizeObserver;
+import org.joda.time.Instant;
+
+/**
+ * A timer consists of a timestamp and a corresponding user supplied payload.
+ *
+ * <p>Note that this implementation is specifically used during execution within runners and inside
+ * the SDK harness. The {@link org.apache.beam.sdk.state.Timer} represents the current user facing
+ * API. Consider consolidating the two once {@link org.apache.beam.runners.core.TimerInternals} is
```

Comment:
This causes a failure in publishing due to a broken link when building `:beam-runners-core-construction:javadoc`. There are two options:

1. Suppress all individual javadoc builds and just build one giant one, so dependencies are a non-issue. This allows references that are often useful, though conceptually "shouldn't" be there.
2. Only talk about things that are actual dependencies, since the library should not require knowledge of thing that are not its dependencies.

This particular sentence seems like it does not add a lot, could be a non-javadoc TODO or a JIRA.

Reply:
I'll swap the comment to not use a `@link`

Reply:
Just noticed that you fixed it, thanks.